com.unity.multiplayer.mlapi/Runtime/Core/NetworkUpdateLoop.cs

```diff
@@ -0,0 +1,525 @@
+﻿using System.Linq;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.LowLevel;
+using UnityEngine.PlayerLoop;
+
+namespace MLAPI
+{
+    /// <summary>
+    /// Defines the required interface of a network update system being executed by the network update loop.
+    /// </summary>
+    public interface INetworkUpdateSystem
+    {
+        void NetworkUpdate(NetworkUpdateStage updateStage);
+    }
+
+    /// <summary>
+    /// Defines network update stages being executed by the network update loop.
+    /// </summary>
+    public enum NetworkUpdateStage : byte
+    {
+        Initialization = 1,
+        EarlyUpdate = 2,
+        FixedUpdate = 3,
+        PreUpdate = 4,
+        Update = 0,
```

Comment:
Why is the zero value for update in the middle of the sequence of values? If it is for a default, or backward compatibility or ..., a comment might be justified

Reply:
it is for default